### DISCUSSION TITLE
[BUG] Get-PnPListItem does not return the list field named 'DisplayName' under FieldValues property

### DISCUSSION THREAD

**Discussion by s-roop** [pnp/powershell#4621] [General]
### Notice
Many bugs reported are actually related to the PnP Framework which is used behind the scenes. Consider carefully where to report an issue:

1. **Are you using ```Invoke-PnPSiteTemplate``` or ```Get-PnPSiteTemplate```**? The issue is most likely related to the Provisioning Engine. The Provisioning engine is _not_ located in the PowerShell repo. Please report the issue here: https://github.com/pnp/pnpframework/issues.
1. **Is the issue related to the cmdlet itself, its parameters, the syntax, or do you suspect it is the code of the cmdlet that is causing the issue?** Then please continue reporting the issue in this repo.
1. **If you think that the functionality might be related to the underlying libraries that the cmdlet is calling** (We realize that might be difficult to determine), please first double check the code of the cmdlet, which can be found here: https://github.com/pnp/powershell/tree/master/src/Commands. If related to the cmdlet, continue reporting the issue here, otherwise report the issue at https://github.com/pnp/pnpframework/issues

### Reporting an Issue or Missing Feature
Please confirm what it is that your reporting:

   When using 'Get-PnPListItem' to get the field values of SharePoint lists, fields with the InternalName of 'DisplayName' are missing from the FieldValues property. 

### Expected behavior 
Please describe what output you expect to see from the PnP PowerShell Cmdlets

    Input: (Get-PnPListItem -List $TestList | Where-Object{$_.FieldValues.Title -eq "Test Row"}).FieldValues.'TestFieldName'
    Output: TestFieldValue
 
### Actual behavior 
Please describe what you see instead. Please provide samples of output or screenshots.

    Input: (Get-PnPListItem -List $TestList | Where-Object{$_.FieldValues.Title -eq "Test Row"}).FieldValues.'DisplayName'
    Output: null/empty value

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)

    $Testlist = Get-PnPList -Identity "Test List"
    (Get-PnPListItem -List $TestList | Where-Object{$_.FieldValues.Title -eq "Test Row"}).FieldValues.'DisplayName'

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)

    2.99.48 nightly

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by ervingayle** [comment]
Hello @s-roop,
Can you try the following?
Typically I would assign the get-pnplistitem to a variable and iterate over each to another array if I needed to do something with the values but this format has worked for me in the past.
$Testlist = Get-PnPList -Identity "Test List"
(Get-PnPListItem -List $TestList -Fields "Title","DisplayName").FieldValues | Where-Object{$_.Title -eq "Test Row"}

**Comment by s-roop** [comment]
> Hello @s-roop, Can you try the following? Typically I would assign the get-pnplistitem to a variable and iterate over each to another array if I needed to do something with the values but this format has worked for me in the past. $Testlist = Get-PnPList -Identity "Test List" (Get-PnPListItem -List $TestList -Fields "Title","DisplayName").FieldValues | Where-Object{$_.Title -eq "Test Row"}

I gave it a try but this only gets the Title field properties - I tried a few more different ways but with no luck.

 I've since created a field with the internal name of 'DisplayName_' and copied the existing data over. Everything works as expected and I've adjusted my scripts for a workaround but it's still an annoying solution.
